cmake/linux/helpers.cmake

```diff
@@ -74,6 +74,8 @@ function(set_target_properties_obs target)
       COMMAND "${CMAKE_COMMAND}" -E make_directory "${OBS_OUTPUT_DIR}/$<CONFIG>/${OBS_LIBRARY_DESTINATION}"
       COMMAND "${CMAKE_COMMAND}" -E copy_if_different "$<TARGET_FILE:${target}>"
               "${OBS_OUTPUT_DIR}/$<CONFIG>/${OBS_LIBRARY_DESTINATION}/"
+      COMMAND "${CMAKE_COMMAND}" -E copy_if_different "$<TARGET_SONAME_FILE:${target}>"
```

Comment:
With the recent change to default to the `SONAME` for all libraries, shouldn't this only copy based on that property and not on the target name to begin with?

Reply:
It all seems a bit chaotic, but that's the entire "copy to rundir" thing to begin with..